opensearch/Dockerfile

```diff
@@ -24,4 +24,12 @@ LABEL git_branch=${GIT_BRANCH}
 LABEL git_commit=${GIT_COMMIT}
 LABEL git_diff=${GIT_DIFF}
 
-CMD ./sycamore-opensearch.sh
+USER root
+RUN yum install -yq openssl jq && yum clean all
+RUN chown 1000:1000 \
```

Comment:
Why are you having to chown here?  They were installed above as 1000.

Reply:
It wasn't explicit in that version, as it was inherited from the parent image.  That seemed error-prone.  Things are rearranged now and I'll try without it.